cloudformation.yml

```diff
@@ -56,4 +56,34 @@ Resources:
         EDLUsername: !Ref EDLUsername
         EDLPassword: !Ref EDLPassword
         RtcGammaImage: !Ref RtcGammaImage
+        ProductBucket: !Ref Bucket
       TemplateURL: batch-cluster/cloudformation.yml
+
+  StepFunction:
+    Type: AWS::CloudFormation::Stack
+    Properties:
+      Parameters:
+        JobQueueArn: !GetAtt Cluster.Outputs.JobQueueArn
+        RtcGammaJobDefinition: !GetAtt Cluster.Outputs.RtcGammaJobDefinition
+      TemplateURL: step-function/cloudformation.yml
+
+  Bucket:
```

Comment:
might call it `ContentBucket` here as well for a more verbose description, since I imagine we'll add `LogBucket` shortly